src/Views/Teams/TeamDetails.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { fetchTeamPlayersById } from '../../services/fetchTeams';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+
+export default function TeamDetails() {
+  const [teams, setTeams] = useState({ players: [] });
```

Comment:
Nice work destructuring players in the useState function so that our data doesn't try to pull from an empty object/array. We can also use loading state here to circumvent the issue.